b2/account_info/sqlite_account_info.py

```diff
@@ -32,6 +32,9 @@ class SqliteAccountInfo(UrlPoolAccountInfo):
     """
     Stores account information in an sqlite database, which is
     used to manage concurrent access to the data.
+
+    The 'update_done' table tracks the schema updates that have been
```

Comment:
I like this very much, but it shouldn't be the part of the `keys` PR. It's an improvement for the next time.

Reply:
It's an improvement for this time, but could have been done before the app keys work.